R/calcFEdemand.R

```diff
@@ -299,16 +299,18 @@ calcFEdemand <- function(subtype = "FE") {
                  scenario = paste0('gdp_', scenario)),
 
         # load VA projections
-        readSource('EDGE_Industry', 'projections_VA_iso3c', convert = FALSE) %>%
-          as.data.frame() %>%
-          as_tibble() %>%
-          select(scenario = Data1, iso3c = Region, year = Year, sector = Data2,
-                 value = Value) %>%
-          filter(grepl('^gdp_SSP[12]$', scenario),
-                 'Total' != iso3c,
-                 as.character(year) %in% years) %>%
-          mutate(iso3c = as.character(iso3c),
-                 year = as.integer(as.character(year))) %>%
+        calcOutput(
+          type = 'Industry_Value_Added',
```

Comment:
where is this new function calcIndustry_Value_Added ? or do I misunderstand you?

Reply:
Here: https://github.com/pik-piam/mrremind/pull/306/files#diff-9807fde4251733b523442dbd38b01184940d083d5946c07b30a85b39710f1976R1237

Reply:
ah, should/could this be a separate calc* or tool* function?